src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -618,26 +618,26 @@ static void window_editor_objective_options_arg_1_decrease(rct_window* w)
 
 static void window_editor_objective_options_arg_2_increase(rct_window* w)
 {
-    if (gScenarioObjectiveYear >= 25)
+    if (gScenarioObjective.Year >= 25)
```

Comment:
I feel all of these limits should also be moved to the class. Not too sure how that would look though. I guess you could go with lots of inheritance but might be overkill.

Reply:
I think the Objective struct should allow more than these limits. In a technical sense, it can take 30 years or 240 guests without any problem, it's just that the Scenario Editor enforces round limits. This is why I think these limits should stay in the Scenario Editor.